src/api/hocrrenderer.cpp

```diff
@@ -26,6 +26,7 @@
 #endif
 #include <tesseract/renderer.h>
 #include "tesseractclass.h" // for Tesseract
+#include "thresholder.h" // for ImageThresholder
```

Comment:
Please use the API instead:

https://github.com/tesseract-ocr/tesseract/blob/60fd2b4abaa9c5c5c42d32db57576bc95d28a78a/src/api/baseapi.cpp#L956-L960

Reply:
Done, thanks.